client/verta/verta/registry/_verta_model_base.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+import abc
+
+from verta.external import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class VertaModelBase(object):
+    """Abstract base class for Verta Standard Models.
+
+    .. note::
+
+        ``__init__()`` and :meth:`predict` **must** be implemented by
+        subclasses. Any other methods are optional.
+
+    Parameters
```

Comment:
Wrong spot?

Reply:
This is the right spot—Sphinx (as far as I've seen) looks for `__init__()` parameters on the class docstring.